features/verblijfplaatshistorie.feature

```diff
@@ -89,41 +89,41 @@ Functionaliteit: Tonen van verblijfplaatsen
     Dan wordt geen enkele verblijfplaats teruggegeven
 
   Scenario: Verblijfplaatsen opvragen over een periode
-    Als de ingeschreven persoon met burgerservicenummer 999994669 wordt geraadpleegd met datumvan 1961-12-29 en datumtotenmet 2019-09-11
+    Als de ingeschreven persoon met burgerservicenummer 999994669 wordt geraadpleegd met datumVan 1961-12-29 en datumTotEnMet 2019-09-11
```

Comment:
In regel 10 staat:

> Ook wanneer de persoon al voor de periode op een verblijfplaats verbleef en binnen de gevraagde periode nog steeds, wordt de verblijfplaats opgenomen in het antwoord. 

moet dit niet zijn:

> Ook wanneer de persoon al voor de periode op een verblijfplaats verbleef en binnen **(een deel van)** de gevraagde periode nog steeds, wordt de verblijfplaats opgenomen in het antwoord.

Reply:
Aangepast